src/components/Quiz.js

```diff
@@ -18,27 +19,40 @@ const Quiz = () => {
     const totalScore = data.reduce((point, question) => {
       const selectedAnswerId = userAnswers[question.id];
       const selectedAnswerObj = question.answers.find(
-        answer => answer.id === Number(selectedAnswerId),
+        answer => answer.id === selectedAnswerId,
       );
       return selectedAnswerObj.isCorrect ? point + 1 : point;
     }, 0);
     setScore(totalScore);
     setShouldDisplayScore(true);
   };
-
+  const question = data[currentQuestionIndex];
```

Comment:
It feels a bit cramped here — adding a newline after (and before) `const question` to give some vertical space makes it a bit nicer to look at imo. Not as important as naming but still nice to have!